tuf/client/updater.py

```diff
@@ -1403,9 +1403,12 @@ def _verify_root_self_signed(self, signable):
       valid_sig = securesystemslib.keys.verify_signature(key, signature, signed)
 
       if valid_sig:
-        validated = validated + 1
+        verified_sig_keyids.append(keyid)
 
-    if validated >= threshold:
+    # A signature with a given keyid should only count towards the threshold
+    # once. Convert the list of keyids into a set, in order to uniquify them
+    # and therefore count each keyid only once towards the threshold.
```

Comment:
```suggestion
    # Each unique signature/keyid counts only once towards the threshold
```
would this be enough?

Reply:
That's much more concise while remaining informative, thanks. I ended up removing the comment entirely when switching to using a set as suggested above.